tests/functional/modules/test_zos_copy_func.py

```diff
@@ -2350,17 +2350,6 @@ def test_copy_ps_to_existing_uss_file(ansible_zos_module, force):
     dest = get_random_file_name(dir=TMP_DIRECTORY)
 
     hosts = ansible_zos_module
-    mlq_size = 3
```

Comment:
Why are you deleting this names?

Reply:
These variables where not being used in the test, I think they got there from a merge error, they appear again in the loadlib tests, where they are actually used